resources/qml/TimelineRow.qml

```diff
@@ -54,6 +54,20 @@ MouseArea {
 			}
 		}
 
+		StatusIndicator {
+			state: model.state
+			Layout.alignment: Qt.AlignRight | Qt.AlignTop
+			Layout.preferredHeight: 16
+			width: 16
+		}
+
+		EncryptionIndicator {
```

Comment:
not sure if it was via this change, but I'm not seeing the encryption indicators in messages.

Reply:
I think you only see encryption indicators for recently sent messages by you. Try sending a message in an encrypted room. If that message does have the indicator, that's an old bug/limitation.

Reply:
tested, confirmed. thanks :)